1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -32,6 +32,14 @@ if [[ `grep CentOS /etc/redhat-release` ]]; then
   MAVEN_SLAVE=${MAVEN_SLAVE_IMAGE:-openshift/jenkins-slave-maven-centos7}
 fi
 
+JENKINS_SERVICE_NAME=${JENKINS_SERVICE_NAME:-JENKINS}
+JENKINS_SERVICE_NAME=`echo ${JENKINS_SERVICE_NAME} | tr '[a-z]' '[A-Z]' | tr '-' '_'`
+
+J_HOST=${JENKINS_SERVICE_NAME}_SERVICE_HOST
+JENKINS_SERVICE_HOST=${!J_HOST}
+
+J_PORT=${JENKINS_SERVICE_NAME}_SERVICE_PORT
+JENKINS_SERVICE_PORT=${!J_PORT}
```

Comment:
should have mentioned to you that v1 is deprecated (i.e. the `1` subdir) ... since the deprecation we have only been updating 2/*